plutus-tx/test/Plugin/Data/Spec.hs

```diff
@@ -89,6 +90,11 @@ recordNewtype = plc @"recordNewtype" (\(x :: RecordNewtype) -> x)
 nonValueCase :: CompiledCode (MyEnum -> Integer)
 nonValueCase = plc @"nonValueCase" (\(x :: MyEnum) -> case x of { Enum1 -> 1::Integer ; Enum2 -> Builtins.error (); })
 
+data StrictPattern a = StrictPattern !a !a
```

Comment:
I can imagine users asking "so... is your language lazy or strict?". We're kind of strict, but then there are those strictness annotations, so this may be confusing.

Reply:
Yeah, it's confusing. However, we can't really ban them, since they're gone by the time we see them. Now we will effectively ignore it, which seems fine.